modules/core/shared/src/main/scala/lucuma/core/validation/InputValidSplitEpi.scala

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.validation
+
+import cats.data.NonEmptyChain
+import cats.syntax.all._
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.api.{Validate => RefinedValidate}
+import eu.timepit.refined.auto._
+import eu.timepit.refined.collection.NonEmpty
+import eu.timepit.refined.numeric.Positive
+import eu.timepit.refined.types.numeric.PosBigDecimal
+import eu.timepit.refined.types.numeric.PosInt
+import eu.timepit.refined.types.string.NonEmptyString
+import lucuma.core.optics._
+import monocle.Iso
+import monocle.Prism
+
+import java.math.RoundingMode
+import java.text.DecimalFormat
+import scala.util.Try
+
+/**
+ * Convenience version of `ValidFormat` when the error type is `NonEmptyChain[NonEmptyString]` and
```

Comment:
Oh, it looks like it was called `ValidFormat` in an earlier version?

Reply:
Yup, see previous comment. Also, good catch, I'm fixing the comment.